resolwe/process/fields.py

```diff
@@ -546,6 +595,10 @@ def _populate_cache(self):
 
     def _get(self, key):
         """Return given key from cache."""
+        # Relations are not stored in self._cache
```

Comment:
Is there a reason why they are not stored in cache? I think it would be sensible as `self.relations` is not a trivial function.

Reply:
Fixed.